j5/boards/zoloto/camera_board.py

```diff
@@ -0,0 +1,45 @@
+"""Virtual Camera Board for detecting Fiducial Markers."""
+
+from typing import Optional, Set, Type, cast
+
+from j5.backends import Backend
+from j5.boards import Board
+from j5.components import Component, MarkerCamera, MarkerCameraInterface
+
+
+class ZolotoCameraBoard(Board):
+    """Virtual Camera Board for detecting fiducial markers."""
+
+    name: str = "Zoloto Camera Board"
+
+    def __init__(self, serial: str, backend: Backend):
+        self._serial = serial
+        self._backend = backend
+
+        self._camera = MarkerCamera(0, cast(MarkerCameraInterface, backend))
+
+    @property
+    def serial(self) -> str:
+        """Get the serial number."""
+        return self._serial
+
+    @property
+    def firmware_version(self) -> Optional[str]:
+        """Get the firmware version of the board."""
+        return None
```

Comment:
You can probably still use the version of `zoloto` here.

Reply:
This is a bit of j5 which needs fixing, there is an issue for it #255. Have implemented the correct interim behaviour.